web/models.py

```
@@ -2805,3 +2805,169 @@
+
+    def save(self, *args, **kwargs):
+        if not self.slug:
+            self.slug = slugify(self.name)
+        super().save(*args, **kwargs)
+
+    @property
+    def member_count(self):
+        return self.members.filter(is_approved=True).count()
+    
+    @property
+    def upcoming_events(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `upcoming_events`

(ANN201)